docs/technical-reference.md

```diff
@@ -550,6 +550,19 @@ Note the comma after the object. Your trust relationship should now look somethi
 
 You can now configure a Quilt Role with this role (using the Catalog's admin panel, or `quilt3.admin.create_role`).
 
+### ManagedUserRoleExtraPolicies
+
+The `ManagedUserRoleExtraPolicies` parameter allows you to add additional IAM policies to the managed user role. This is useful for granting additional permissions to users in your Quilt instance, which otherwise would be blocked by the permission boundary.
```

Comment:
IMO it's not quite clear that you need both to modify `ManagedUserRoleExtraPolicies` and add policy in Quilt Admin

Reply:
This applies to more than just User Policies (see KMS below).  I will add clarifying text, though.

Reply:
I believe that currently `ManagedUserRoleExtraPolicies` affects only "managed" user role, in KMS section permissions are granted to other roles with resource policy

Reply:
Sorry, I'm not understanding your comment.  I think my PR is extending the current documentation in a correct way.  Am I mis-stating something, or is there a flaw in the original documentation we also need to fix?

Reply:
see my top comment

> This applies to more than just User Policies (see KMS below)

that seems both wrong and irrelevant
KMS permissions are granted to "managed" user roles in the steps 1 and 3, and to other roles in the step 2, these "other" roles don't need step 1
so if you want to extract info about granting permissions to "managed" user roles, you need to extract both steps 1 and 3

Reply:
Does this address your concern?